chapters/functions.tex

```diff
@@ -1928,6 +1928,7 @@ \subsubsection{Arrays}\label{arrays-1}
 
 The table below shows the mapping of an array argument in the absence of an explicit external function call when calling a C function.
 The type {\lstinline!T!} is allowed to be any of the simple types which can be passed to C as defined in \cref{simple-types} or a record type as defined in \cref{records} and it is mapped to the type $T'$ as defined in these sections for input arguments.
+(Which for array of records mean passing a (possibly \lstinline[language=C]!const!) pointer to the first element of an array of structs in C.)
```

Comment:
I don't think there should be any need for clarification here.  We should just be more clear in the section on records.
```suggestion
```